swap_meet/vendor.py

```diff
@@ -1,2 +1,73 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    
+    def __init__(self, inventory = None):
+        self.inventory = inventory if inventory is not None else []
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
+
+    def get_by_category(self, category):
+        # alternative solution: 
+        # list_of_items = []
+        # for item in self.inventory:
+        #     if item.category == category:
+        #         list_of_items.append(item)
+        # return list_of_items
+
+        list_of_items = [item for item in self.inventory 
+        if item.category == category]
+        return list_of_items
+
+    def swap_items(self, friend, my_item, their_item):
+        if my_item not in self.inventory or their_item not in friend.inventory:
+            return False
+
+        item_being_removed = self.remove(my_item)
+        friend.add(item_being_removed)
+        print(self.inventory, friend.inventory)
```

Comment:
Let's get rid of print statements that are for debugging now that we have submitted our final version.

```suggestion
```